src/main/java/com/jcabi/github/RtReleases.java

```diff
@@ -64,6 +65,16 @@
     private final transient Request request;
 
     /**
+     * RESTful API releases request.
+     */
+    private final transient Request latest;
```

Comment:
@weisebrazil you don't need this field, it can be created during each `latest()` method call